charts/argo-cd/values.yaml

```diff
@@ -1889,11 +1890,14 @@ applicationSet:
 
   image:
     # -- Repository to use for the application set controller
+    # @default -- `quay.io/argoproj/argocd-applicationset` (defaults to global.image.repository)
```

Comment:
Can we remove this line?

Reply:
@mkilchhofer - we can remove this but implementation internally really defaults to global config